dev/test/write-batch.ts

```diff
@@ -76,6 +78,24 @@ describe('set() method', () => {
     nullObject.bar = 'ack';
     writeBatch.set(firestore.doc('sub/doc'), nullObject);
   });
+
+  it('requires toFirestoreFromMerge() for Partial usage', () => {
+    const converter = postConverter;
+    delete converter.toFirestoreFromMerge;
```

Comment:
This likely breaks the tests if they are run out of order since you are modifying an instance that is used in other tests.

Reply:
Tried to be a cool 2020 kid and used the spread operator.